src/sources/journald.rs

```diff
@@ -177,6 +236,89 @@ where
     }
 }
 
+const CHECKPOINT_FILENAME: &'static str = "checkpoint.txt";
```

Comment:
would it make sense to name this something related to journald?

Reply:
Sure. Any suggestions? `journald-checkpoint.txt`?